### ISSUE TITLE
Blog page accessibility issues

### ISSUE THREAD

**Issue by gixxerblade** [Vets-Who-Code/vets-who-code-app#224]
# Accessibility

## Background and foreground colors do not have a sufficient contrast ratio.

Low-contrast text is difficult or impossible for many users to read. [Learn More](https://web.dev/color-contrast/?utm_source=lighthouse&utm_medium=devtools)

### Failing Elements

- by Jerome Hardaway `<span class="author" style="padding-right:4px"> <!-- -->by<!-- --> <span class="blog-author" style="outline:none">Jerome Hardaway</span></span>`
- May 7th, 2020 `<time class="entry-date" datetime="">`
- by Jerome Hardaway `<span class="author" style="padding-right:4px"> <!-- -->by<!-- --> <span class="blog-author" style="outline:none">Jerome Hardaway</span></span>`
- May 5th, 2020 `<time class="entry-date" datetime="">`
- by Jerome Hardaway `<span class="author" style="padding-right:4px"> <!-- -->by<!-- --> <span class="blog-author" style="outline:none">Jerome Hardaway</span></span>`
- April 14th, 2020 `<time class="entry-date" datetime="">`

**To Reproduce**
1. Go to 'https://vetswhocode.io/blog'
2. Click on 'Developer Tools (F12)'
3. Click on Lighthouse
4. Generate report

**Expected behavior**
Low contrast ratio - Text that has a low contrast ratio can be hard to read. A contrast ratio of at least 4.5:1 or greater is recommended.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by joneskj55** [comment]
@gixxerblade I'd like to take this if possible.

**Comment by gixxerblade** [comment]
Hi @joneskj55. Unfortunately, due to environment variable sharing, this issue has to be taken by one of the team internally. You are welcome to tackle #215 which covers heading elements that are not in a sequentially-descending order on our homepage. Thank you so much for contributing!

**Comment by joneskj55** [comment]
Thanks @gixxerblade I'll take #215. Happy to help out!

**Comment by gixxerblade** [comment]
Closed with #234
